scripts/compare-api-responses/index.mjs

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { Command } from 'commander'
+import { diffLinesUnified } from 'jest-diff'
+import { PRODUCT_CONFIG as contentDirMap } from '../../app/utils/productConfig.mjs'
+import stripAnsi from 'strip-ansi'
+import fs from 'node:fs'
+import path from 'node:path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url) // get the resolved path to the file
+const __dirname = path.dirname(__filename) // get the name of the directory
+
+const versionMetadataPath = path.resolve(
+	__dirname,
+	'../../app/api/versionMetadata.json',
+)
+const versionMetadata = JSON.parse(
+	fs.readFileSync(versionMetadataPath, 'utf-8'),
+)
+
+const docsPathsAllVersionsPaths = path.resolve(
+	__dirname,
+	'../../app/api/docsPathsAllVersions.json',
+)
+const apiPaths = JSON.parse(fs.readFileSync(docsPathsAllVersionsPaths, 'utf-8'))
+
+const program = new Command()
+
+program
+	.option('-n, --new-api-url <url>', 'New API URL')
+	.option('-o, --old-api-url <url>', 'Old API URL')
```

Comment:
@heatlikeheatwave let's set the default here to be our current content API: `https://content.hashicorp.com`

Reply:
done https://github.com/hashicorp/web-unified-docs/pull/232/commits/ecb376ea0ae98025f57acc70033e773ba7920c42